babel.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
     '@vue/cli-plugin-babel/preset'
   ],
   plugins: ["@babel/plugin-proposal-private-methods",
+    ["babel-plugin-root-import", { "rootPathSuffix": "./", "rootPathPrefix": "@/"}],
```

Comment:
Do we need this back in, now that we have `module-resolver`?

Reply:
Status bar issue: https://github.com/mattfarina/rd/issues/90

And the `babel-plugin-root-import` field got dropped in a different branch.